arquillian-ape-sql/core-dbunit/src/main/java/org/arquillian/ape/rdbms/dbunit/DbUnitPopulatorService.java

```diff
@@ -109,6 +109,11 @@ public void clean(List<String> resources) {
     private Optional<IDatabaseConnection> lookupDataSourceConnection(String dataSourceName) {
 
         try {
+
+            if (contextInstance == null) {
+                // In case of running outside Arquillian Runner
```

Comment:
Hmmm this is getting a bit ugly :) Having classes based on ARQ services / concepts used outside of its lifecycle we will end up with a lot of "ignore if `null`" - which is not really the nicest thing to have.